src/serde_arrow_array.erl

```diff
@@ -86,7 +86,7 @@
 -include("serde_arrow_array.hrl").
 
 -export_type([layout/0]).
--type layout() :: primitive.
+-type layout() :: primitive | binary.
```

Comment:
```suggestion
-type layout() :: fixed | variable.
```

?

Reply:
So maybe `fixed_primitive | variable_binary`? Which will pair nicely with `fixed_list` and `variable_list`?